### PR TITLE
Lint.py: Replace deprecated link

### LINKED ISSUE #2774: Remove deprecated link in the document string from the `Lint` class in the `Lint.py` file
http://coala.readthedocs.io/en/latest/Users/Tutorials/Linter_Bears.html is the link in the file, it is 404'd, it needs to be updated.

Complete path of the file : `coala/coalib/bearlib/abstractions/Lint.py`

difficulty/newcomer

### PR DESCRIPTION
**Pull request by Dawny33**
Replaced the deprecated link from the `Lint` class in `Lint.py`

Replaced the (deprecated)link "http://coala.readthedocs.io/en/latest/Users/Tutorials/Linter_Bears.html" in the `Lint` class' docstring in `Lint.py` to "https://coala.readthedocs.io/en/stable/Users/Tutorials/Linter_Bears.html"

Closes #2774

### CODE REVIEW

File: coalib/bearlib/abstractions/Lint.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ class Lint(Bear):
     Deals with the creation of linting bears.
 
     For the tutorial see:
-    http://coala.readthedocs.org/en/latest/Users/Tutorials/Linter_Bears.html
+    https://coala.readthedocs.io/en/stable/Users/Tutorials/Linter_Bears.html
```
Comment (sils1297) [inline coalib/bearlib/abstractions/Lint.py]: rather use docs.coala.io instead of coala.readthedocs.io, as it's shorter (does the same)

**Comment by sils1297** [comment]
Hi,

it seems that your commit message doesn't contain enough information. Don't worry, most people don't know about good commit messages when they join us, including very experienced developers :) Standards vary from project to project.

The most important part is the first line of your commit message - maintainers will use it to read through a large number of commits and thus it's important that it's short but most importantly concise. I'd heartily recommend reading <http://coala.io/commit>!

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix up the message for you - just tell us :).

Cheers!

**Comment by sils1297** [comment]
hey, you'll need to do another amend, the body is too long, see coala.io/commit again (74 chars max I think, vim dos it automatically for me :P)

**Comment by sils1297** [comment]
Also, it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala/coala.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

**Comment by sils** [comment]
Hi,

do you need help figuring this out? We're all happy to help you on [our gitter channel](https://coala.io/chat), just drop by and ask us anything :)

Should someone else take this over for you?

**Comment by aptrishu** [comment]
@Dawny33  You have lots of PRs pending please see to it. Please ask us if you need any help.
feel free to ping us on our [gitter channel.](https://gitter.im/coala/coala).

**Comment by Makman2** [comment]
Lint is deleted by now, link has also changed, PR invalid.

**Pull request closed without merging**